src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -49,7 +49,7 @@ class RsFileStub(
     override fun getType() = Type
 
     object Type : IStubFileElementType<RsFileStub>(RsLanguage) {
-        private const val STUB_VERSION = 206
+        private const val STUB_VERSION = 207
```

Comment:
```suggestion
        private const val STUB_VERSION = 208
```

The current version in master is already 207